test/proposals/proposals.jl

```diff
@@ -32,3 +33,8 @@ end
 
 @testset "RWalk" begin
 end
+
+@testset "RStagger" begin
+    # printing
+    @test Proposals.RStagger(;ratio=0.5, walks=25, scale=1) isa NestedSamplers.Proposals.RStagger
```

Comment:
Good idea adding a test here!

Unfortunatley, I wouldn't say this test is too useful- I would be extremely surprised if after I made an object if the object changed types! I think some more useful tests would be along the lines of _are the values being set correctly?_

Perhaps you could add some tests like this-
```julia
# Test defaults
prop = Proposals.RStagger()
@test prop.ratio == 0.5
# and so on...

prop2 = Proposals.RStagger(ratio=0.1, walks=20)
@test prop2.ratio == 0.1
# and so on...
```

Reply:
I think you should include both -- this test is a good smoke test to make sure that the constructor is returning what you want it to return. It's actually very useful here because the test is actually failing: https://github.com/TuringLang/NestedSamplers.jl/pull/38/checks?check_run_id=808297816

Reply:
I agree, it's not bad to include both

that test failure is a different test, though.

Reply:
Yeah, saw that after this. My bad.